src/logs.ts

```diff
@@ -131,40 +142,27 @@ export class Logs {
     Logs.console = new PrettyLogs();
   }
 
-  private _diffColorCommentMessage(type: string, message: string) {
-    const diffPrefix = {
-      fatal: "-", // - text in red
-      ok: "+", // + text in green
-      error: "!", // ! text in orange
-      info: "#", // # text in gray
-      debug: "@@@@", // @@ text in purple (and bold)@@
+  private _diffColorCommentMessage(type: LogLevelWithOk, message: string) {
+    const diffPrefix: Record<LogLevelWithOk, string> = {
+      fatal: "> [!CAUTION]",
+      error: "> [!CAUTION]",
+      warn: "> [!WARNING]",
+      ok: "> [!NOTE]",
+      info: "> [!NOTE]",
+      debug: "> [!NOTE]",
```

Comment:
Seems wrong to have so many redundant. Also, this one should be the purple one.

Reply:
so I should use `[!IMPORTANT]` for debug? I put `[!NOTE]` because important and tip didn't seem appropriate

Reply:
Right now I'm thinking the title is less important compared to the message contents. Let's do purple for this one and see how it plays out in practice.